JapaneseFilter/sections/specific.txt

```diff
@@ -4606,8 +4606,10 @@ kobe-np.co.jp##.prNews
 kobe-np.co.jp##div[id^="knpc-"]
 st.benesse.ne.jp##div[id^="div-M_MD_kiji_fluid_"]
 benesse.ne.jp##div[style^="height: 250px"]
+39mag.benesse.ne.jp##.recommend
```

Comment:
We have `estate.nikkan.co.jp,marieclairejapon.com,daily.co.jp,post.tv-asahi.co.jp,milk-key.com,fujinkoron.jp##.recommend`

Reply:
sorry